src/index/hnsw/faiss_hnsw.cc

```diff
@@ -357,21 +362,27 @@ convert_rows_from_fp32(const float* const __restrict src, void* const __restrict
         for (size_t i = 0; i < nrows * dim; i++) {
             dst[i + start_row * dim] = (knowhere::fp16)src[i];
         }
-
         return true;
     } else if (dst_data_format == DataFormatEnum::bf16) {
         knowhere::bf16* const dst = reinterpret_cast<knowhere::bf16*>(dst_in);
         for (size_t i = 0; i < nrows * dim; i++) {
             dst[i + start_row * dim] = (knowhere::bf16)src[i];
         }
-
         return true;
     } else if (dst_data_format == DataFormatEnum::fp32) {
         knowhere::fp32* const dst = reinterpret_cast<knowhere::fp32*>(dst_in);
         for (size_t i = 0; i < nrows * dim; i++) {
             dst[i + start_row * dim] = src[i];
         }
-
+        return true;
+    } else if (dst_data_format == DataFormatEnum::int8) {
+        knowhere::int8* const dst = reinterpret_cast<knowhere::int8*>(dst_in);
+        for (size_t i = 0; i < nrows * dim; i++) {
+            KNOWHERE_THROW_IF_NOT_MSG(src[i] >= std::numeric_limits<knowhere::int8>::min() &&
```

Comment:
it is better to use `std::numeric_limilts<knowhere::int8>::lowest()` here

Reply:
Hi Alex, what's the difference between min() and lowest() here?

Reply:
there's no difference for this particular use case. But `lowest()` is better to use, because of the connotations with `std::numeric<float>::lowest()` (which is -1e+40, the least value) and `std::numeric<float>::min()` (which is 1e-40, the least representable positive value)